assets/javascripts/discourse/components/ai-persona-list-editor.gjs

```diff
@@ -21,45 +27,93 @@ export default class AiPersonaListEditor extends Component {
     return this._noPersonaText;
   }
 
+  @action
+  async toggleEnabled(persona) {
+    const oldValue = persona.enabled;
+    const newValue = !oldValue;
+
+    try {
+      persona.set("enabled", newValue);
+      await persona.save();
+    } catch (err) {
+      persona.set("enabled", oldValue);
+      popupAjaxError(err);
+    }
+  }
```

Comment:
Is this the canonical way of doing this? It's straight-forward, but also feels like there should be some built-in for this. 🤔

Reply:
Yeah the toggle is only relatively recent, it was first used in the plugin list. We can certainly extract out into a component at some point :+1: